exporters/trace/zipkin/zipkin_test.go

```diff
@@ -389,3 +385,40 @@ func TestErrorOnExportShutdownExporter(t *testing.T) {
 	assert.NoError(t, exp.Shutdown(context.Background()))
 	assert.NoError(t, exp.ExportSpans(context.Background(), nil))
 }
+
+func TestNewExportPipelineWithOptions(t *testing.T) {
+	const eventCountLimit = 10
+
+	collector := startMockZipkinCollector(t)
+	defer collector.Close()
+
+	tp, err := NewExportPipeline(collector.url,
+		WithSDKOptions(
+			sdktrace.WithResource(resource.NewWithAttributes(
+				semconv.ServiceNameKey.String("zipkin-test"),
+			)),
+			sdktrace.WithSpanLimits(sdktrace.SpanLimits{
+				EventCountLimit: eventCountLimit,
+			}),
+		),
+	)
+	require.NoError(t, err)
+
+	otel.SetTracerProvider(tp)
+	_, span := otel.Tracer("zipkin-tracer").Start(context.TODO(), "test-span")
+	for i := 0; i < eventCountLimit*2; i++ {
+		span.AddEvent(fmt.Sprintf("event-%d", i))
+	}
+	span.End()
+
+	err = tp.ForceFlush(context.TODO())
+	require.NoError(t, err)
+
+	checkFunc := func() bool {
+		return collector.ModelsLen() == 1
+	}
+	// It should wait for more than a default batch timeout of BatchSpanProcessor since Zipkin uses default options.
+	require.Eventually(t, checkFunc, 10*time.Second, 10*time.Millisecond)
```

Comment:
The shutdown (or force flush) above should flush the exporter and not require this wait here. If this does indeed require a wait it will introduce a bug on slow CI systems. Can we please verify this is not needed and remove it? Or replace it with a deterministic signal of some sort?

Reply:
My comment is incorrect. The BatshSpanProcessor exports the span correctly when shutting down. Only need time for `mockZipkinCollector` to receive it. 
I change the test time to a second as other tests.